resources/views/admin/events/suggestions.blade.php

```diff
@@ -1,3 +1,4 @@
+@php use Illuminate\Support\Facades\Date; @endphp
```

Comment:
do you like this? I try to don't import things in blade, because I find it ugly. 👀

Reply:
Yes, I like this. I hate FQDNs. :D